app/controllers/application_controller.rb

```diff
@@ -325,12 +337,6 @@ def redirect_or_add_flash
     end
   end
 
-  def redirect_to_getyourrefund
-    if request.get? && request.host.include?("vitataxhelp.org")
```

Comment:
we don't have the app on vitataxhelp.org at all anymore, so this code is totally stale.